### DISCUSSION TITLE
Integration of Shelly Plus Plug S

### DISCUSSION THREAD

**Discussion by TJ23456** [camueller/SmartApplianceEnabler#581] [Allgemein]
Hi,
because the Shelly Plus Plug S are still not supported natively in SMA SM2.0 I am using SAE with these Plugs. I would like to share my working configuration - maybe it is worth to take this over into the SAE-Shelly-doc. In general it is straight forward to other Shelly-V2 Power-devices.

![ShellyPlusPlugS-1](https://github.com/camueller/SmartApplianceEnabler/assets/151472104/71414c5c-76c0-4d1b-bbcd-262f4fbd2728)
![ShellyPlusPlugS-2](https://github.com/camueller/SmartApplianceEnabler/assets/151472104/b2c108c1-4de6-49b9-843c-8d8a1469015b)
![ShellyPlusPlugS-3](https://github.com/camueller/SmartApplianceEnabler/assets/151472104/18061474-bbb5-4697-a2ec-e95292762f29)

Works well in Sunny-Portal:
![Einplanung im SM2 0](https://github.com/camueller/SmartApplianceEnabler/assets/151472104/2f6c87b0-dd08-4a67-9201-1c9a3363eb65)
![SM2 0 Energiezählung](https://github.com/camueller/SmartApplianceEnabler/assets/151472104/b8d25f92-f493-4d8a-b552-4b82a1b28ed2)

Thanks for working on SAE!

**Comment by camueller** [comment]
I included it in the documentation.
